Clients/GUI/Pages/SignUp.razor

```diff
@@ -0,0 +1,59 @@
+﻿@page "/signup"
+@using GUIModels
+@using GUI.Scripts
+@inject NavigationManager NavigationManager
+
+    <div class="col-12">
+        <h3><b>Sign up</b></h3>
+        <hr />
+
+        <EditForm Model="@userInput" OnValidSubmit="@TrySignUp">
+            <DataAnnotationsValidator />
+            <div class="form-group">
+                <label for="Email">Email</label>
+                <InputText class="form-control col-3" @bind-Value="userInput.Email" placeholder="example@gmail.com" />
+                <label style=“color:red;display:@isErrorDisplayed”> @errorMessage </label>
+            </div>
+            <br />
+            <div class="form-group">
+                <label for="Password">Password</label>
+                <InputText class="form-control col-3" type="password" @bind-Value="userInput.Password" placeholder="*****" />
+            </div>
+            <br />
+            <div class="form-group">
+                <label for="ConfirmPassword">Confirm Password</label>
+                <InputText class="form-control col-3" type="password" @bind-Value="userInput.ConfirmPassword" placeholder="*****" />
+            </div>
+            <br />
+            <ValidationSummary />
+            <br />
+
+            <input type="submit" class="form-control col-1 button btn-primary" value="Sign up" />
+
+        </EditForm>
+    </div>
+
+
+@code {
+    bool isErrorDisplayed = false;
+    string errorMessage;
+    private SignUpViewModel userInput;
+
+    public void TrySignUp()
+    {
+        var statusCode = SignUpper.SignUp(userInput);
+        if (statusCode == System.Net.HttpStatusCode.OK)
+            NavigationManager.NavigateTo("/signin");
```

Comment:
why do we need go on signin page, why not on userinfo page?

Reply:
Because user has just registered. If he wants to go to userinfo page, he must log in